src/AspNet.Security.OAuth.Etsy/EtsyAuthenticationConstants.cs

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+
+namespace AspNet.Security.OAuth.Etsy;
+
+/// <summary>
+/// Contains constants specific to the <see cref="EtsyAuthenticationHandler"/>.
+/// </summary>
+public static class EtsyAuthenticationConstants
+{
+    /// <summary>
+    /// Contains claim type constants specific to Etsy authentication.
+    /// </summary>
+    public static class Claims
+    {
+        /// <summary>The claim type for the user's Etsy shop ID.</summary>
+        public static readonly string ShopId = "urn:etsy:shop_id";
+
+        /// <summary>The claim type for the user's profile image URL.</summary>
+        public static readonly string ImageUrl = "urn:etsy:image_url";
+    }
+
+    /// <summary>
+    /// Contains the <see href="https://developers.etsy.com/documentation/reference#section/Authentication/oauth2">Etsy OAuth Scopes</see> constants for Etsy authentication.
```

Comment:
As new ones might be added to the API in the future that aren't in the provider:

```suggestion
    /// Contains <see href="https://developers.etsy.com/documentation/reference#section/Authentication/oauth2">Etsy OAuth Scopes</see> constants for Etsy authentication.
```